Makefile

```
@@ -125,6 +125,14 @@ cmd/bin/subctl-%: $(shell find . -name "*.go")
 		-X 'github.com/submariner-io/submariner-operator/api/v1alpha1.DefaultRepo=$(DEFAULT_REPO)'" \
 	$(SCRIPTS_DIR)/compile.sh $@ ./cmd
 
+cmd/bin/non_deploy_subctl:
+	mkdir -p cmd/bin
```

Let’s use some Make variables to avoid repeating stuff:

```suggestion
	mkdir -p $(@D)
```

---

Should I update L116 as well?

---

Ah yes, let’s do that, thanks!